content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -59,10 +59,16 @@ export class _StartupOverlay extends React.PureComponent {
   }
 
   render() {
+    // When skipping the onboarding tour we show AS but we are still on
+    // about:welcome, prop.isFirstrun is true and StartupOverlay is rendered
+    if (!this.state.show) {
+      return null;
+    }
+
     let termsLink = (<a href="https://accounts.firefox.com/legal/terms" target="_blank" rel="noopener noreferrer"><FormattedMessage id="firstrun_terms_of_service" /></a>);
     let privacyLink = (<a href="https://accounts.firefox.com/legal/privacy" target="_blank" rel="noopener noreferrer"><FormattedMessage id="firstrun_privacy_notice" /></a>);
     return (
-      <div className={`overlay-wrapper ${this.state.show ? "show " : ""}`}>
+      <div className="overlay-wrapper show">
```

Comment:
Isn't the non-show overlay-wrapper needed so that there's the transition? @ericawright ?